File: src/src_platform_swing/fmg/swing/draw/img/StaticInitilizer.java
package fmg.swing.draw.img;

import javax.swing.SwingUtilities;

import fmg.common.notyfier.NotifyPropertyChanged;
import fmg.core.img.AnimatedImgController;

public final class StaticInitilizer {

   static {
      NotifyPropertyChanged.DEFERR_INVOKER = doRun -> SwingUtilities.invokeLater(doRun);
      AnimatedImgController.GET_ANIMATOR = () -> Animator.getSingleton();
   }

   public static void init() {
      // implicit call static block
   }

}

File: src/src_platform_uwp/fmg/uwp/utils/StaticInitilizer.cs
using Windows.UI.Core;
using fmg.core.img;
using fmg.uwp.utils;

namespace fmg.uwp {

   public static class StaticInitilizer {

      static StaticInitilizer() {
         ImageModelConsts.DeferrInvoker = doRun => AsyncRunner.InvokeFromUiLater(() => doRun(), CoreDispatcherPriority.Normal);
         RotatedImgConst.TimerCreator = () => new Timer();
      }

      public static void Init() {
         // implicit call static block
      }

   }

}
